galite-tests/src/test/kotlin/org/kopi/galite/tests/ui/vaadin/block/SimpleBlockTests.kt

```diff
@@ -65,6 +69,76 @@ class SimpleBlockTests: GaliteVUITestBase() {
     assertEquals("123", value)
   }
 
+  @Test
+  fun `test timeStamp field value is sent to model`() {
+    // Login
+    login()
+
+    // Open client form
+    modulesMenu._clickItemWithCaptionAndWait("Bills form")
+
+    // Find fields
+    val fields = _find<TextField> { classes = Styles.TEXT_FIELD }
+    val addressField = fields[0]
+    val dateField = fields[1]
+
+    dateField._clickAndWait(100)
+    // set field value
+    dateField._value = "2018-09-13 00:00:00"
+    val inputField = dateField.field.content as AbstractField<*, String?>
+    inputField._fireEvent(AbstractField.ComponentValueChangeEvent(inputField, inputField, "2018-09-13 00:00:00", true))
+
+    // Focus on another field
+    addressField._clickAndWait(100)
+
+    // Assert the value is sent to the model
+    val value = dateField.model.getTimestamp().toString()
+    assertEquals("2018-09-13 00:00:00.000000", value)
+  }
+
+  @Test
+  fun `test boolean field value is sent to model`() {
+    // Login
+    login()
+
+    // Open client form
+    modulesMenu._clickItemWithCaptionAndWait("Client form")
+
+    val simpleBlock  = _get<SimpleBlockLayout> { classes = "simple"}
+
+    // Find fields
+    val fields = _find<TextField> { classes = Styles.TEXT_FIELD }
+    val codeField = fields[0]
+
+    simpleBlock._get<DBooleanField> {}.wrappedField.isVisible = true
```

Comment:
`simpleBlock._get<DBooleanField>()` instead of `simpleBlock._get<DBooleanField> {}`